src/hooks/useSidePanelHandler.ts

```diff
@@ -0,0 +1,21 @@
+/* External dependencies */
+import { useCallback } from 'react'
+import { LayoutActions } from '../layout/redux'
+
+/* Internal dependencies */
+import useLayoutDispatch from './useLayoutDispatch'
+import useLayoutState from './useLayoutState'
+
+export default function useSidePanelHandler() {
+  const dispatch = useLayoutDispatch()
+  const { showSidePanel } = useLayoutState()
+
+  const handleOpenSidePanel = useCallback(() => dispatch(LayoutActions.setShowSidePanel(true)), [dispatch])
+  const handleCloseSidePanel = useCallback(() => dispatch(LayoutActions.setShowSidePanel(false)), [dispatch])
+
+  return [
+    showSidePanel,
+    handleOpenSidePanel,
+    handleCloseSidePanel,
+  ] as [boolean, () => void, () => void]
```

Comment:
tuple 처리하는게 관리포인트가 줄지 않을까 싶습니다
```suggestion
  ] as const
```